core/src/main/java/com/linecorp/armeria/server/logging/AccessLogComponent.java

```diff
@@ -277,8 +277,7 @@ public Object getMessage0(RequestLog log) {
                     } else {
                         // %a means the client address who initiated a request.
                         final ServiceRequestContext ctx = (ServiceRequestContext) log.context();
-                        return ctx.clientAddressMapper().apply(ctx.proxiedAddresses())
-                                  .getHostString();
+                        return ctx.clientAddress().getHostAddress();
```

Comment:
I didn't mean you have to revert. I meant port should not be included. Could you revert the revert?

Reply:
Oops 😅. I have a question about `clientAddressMapper()`. `AccessLogComponent` is difficult to access `ServerConfig` in the current API because `CommonComponent` is made statically.
https://github.com/line/armeria/blob/15decdfbc881ee0e3b12ca13f396010a54e39ed2/core/src/main/java/com/linecorp/armeria/server/logging/AccessLogFormats.java#L82-L83
 
So I was thinking about two options. 
1) Add `clientAddressMapper()` to `ServiceRequestContext`
2) Add `clientAddressMapper()` to `DefaultRequestLog`

Do I choose the second option? Or if you have another idea, let me know. 😊

Reply:
You can just use `ServiceRequestContext.clientAddress()` because you applied the mapper there already.

Reply:
> Could you revert the revert?

Had chat about `revert the revert` and we talked that the current code(reverted version) is fine. 😀